### ISSUE TITLE
Who has the game 'checked out'?

### ISSUE THREAD

**Issue by KVonGit** [KVonGit/HVACR-specifications#9]
# If someone has the game checked out, and you want to add something on, just post a comment here to alert whoever that particular individual is.

## If I have it checked out, I'll upload what I've currently got and pass it on over.

- [ ] DL

- [x] KV

- [ ] XM

- [ ] no one

Current version: 0.0.91

Quick notes: You can check/uncheck these without editing the comment.

**Comment by DarkLizerd** [comment]
I'm checking it back in... I'll work on something later...
(Gota see what up for now)

**Comment by KVonGit** [comment]
Check it out, peoples!

I'm done (except for cleaning up typos and glitches)!

https://github.com/KVonGit/HVACR-specifications/raw/master/refrigerant%20charts/4px.003.quest

---
If my part is too long, I can whittle it all down and release Pixie's Lament as a separate game. No sweat.

**Comment by DarkLizerd** [comment]
Bugs...
in xan music studio,
put drive in slot.... Doesn't work
use drive on slot.... works, but would not be my first choice to use this.
play _instruments_    gives a syntax error...
play drums     works
use drums      does not work.
After wearing the headset, the room should be quieter...
You should not be able to use the portal while wearing the headset, "the cord is too short."
If the music is so loud, trying to do anything other than "get headset" and "wear headset" should return a "It is too loud to even constrate on doing that."
Give drive to xan... but it is still in the inventory.


I fixed my disk, the globe can now be turned on and off.
Ya want a Wumpus... I've got an idea...

**Comment by KVonGit** [comment]
>put drive in slot.... Doesn't work
use drive on slot.... works, but would not be my first choice to use this.

I ran into this one, too.

>play instruments gives a syntax error...
play drums works
use drums does not work.

I notice the 'play instruments' but thought maybe XM may be adding more instruments...

>After wearing the headset, the room should be quieter...

We could make a duplicate audio track with lowered volume and switch tracks when the headphones are put on... (You'd be able to hear the song start back over though... It wouldn't be a smooth transition...)

>You should not be able to use the portal while wearing the headset, "the cord is too short."

This is a good idea. (It's what I do when the player is holding a corded phone. (The headset isn't cordless, is it?))

>If the music is so loud, trying to do anything other than "get headset" and "wear headset" should return a "It is too loud to even concentrate on doing that."

```C++
if (not EndsWith(player.currentcommand, "headphones") {
  msg ("It is too loud to even concentrate on doing that.")
}
```

NOTE: If the player entered GET HEAD rather than GET HEADPHONES (which is what I entered to get the headphones), I don't think this would work.


>Give drive to xan... but it is still in the inventory.

I think he hands it back to you, but I may be mistaken...


>I fixed my disk, the globe can now be turned on and off.

Rock and roll!

>Ya want a Wumpus... I've got an idea...

WUMPUS!
WUMPUS!
WUMPUS!

>Did you find THE GOD DEVICE in my dimension? Did you EXPERIENCE The God Device?

We could add WUMPUS 2.0 just like I did The God Device.

When the player enters HUNT THE WUMPUS for the first time (to start the game within the game), it could add the WUMPUS to play in the real game!

**Comment by KVonGit** [comment]
@DarkLizerd 

Check this DIFF out and see if it includes every change you recently made, please, sir:

https://github.com/KVonGit/HVACR-specifications/commit/ebf23f550131271711af95da21663cad1454b8a7#diff-efc07a3c44f525e3807f477bc8d3aeb9

I think I committed an update just after you downloaded the file, so now both files have different revisions. (I think I updated it without 'checking it out' first... Whoops! Sorry!)

If you only edited things in your room, I can fix it easily.

Sorry again, sir!

**Comment by XanMag** [comment]
USB port:
Again, rushed through it just to get what was in my head out so you can see it.  Put Drive in slot will work on my next update.

Instruments:
I added a play verb but didn't set up a response with my script.  Rushed it to get basic idea out there.  This has been fixed and will work as intended once update.

Headphones:
They are noise canceling headphones so you can hear nothing when you put them on.  They are there, and plugged in, so that you can hear yourself play the instruments you are playing (the drums). As for leaving the room with them on, I do have it set so that they are removed when you leave the room (not sure if that's the case with the rushed upload I did but it is now). I figured not allowing him to leave the room was an unnecessary step, but I could have a message print that says "you remove the headphones before stepping through the portal" if you think it is that important.

Flash drive:
You are correct.  If you hand him a blank drive, he listens, and he hands it back in the same message.

**Comment by KVonGit** [comment]
>Headphones:
They are noise canceling headphones so you can hear nothing when you put them on.

Damn and blast! (I knew that! I just haven't played with your audio files intact yet. (Although I did download them....))


---
And I've got some 'filler/placeholders' in my parts, too...

Like the help thingy I just made... It needs to be scaled down.

**Comment by KVonGit** [comment]
I just committed an update to the main game file.

**Comment by KVonGit** [comment]
> rushed through it just to get what was in my head out so you can see it.

I'm glad I'm not the only one who does that.

**Comment by DarkLizerd** [comment]
My fix was a small tweak in my area only...
It would be easy to update.
Wumpus 3.0 would be a better tie-in than 2.0...
(Now, I need to finish that upgrade...)

**Comment by KVonGit** [comment]
>My fix was a small tweak in my area only...
It would be easy to update.

If it was just your area, we're all gravy!

I copied the entire section of DL_room from your code and pasted it over the same section in the main code.

---

I just made all the comments on the commit because I was trying to sort out all the changes to make sure I didn't fuck anything up.

I didn't know it would alert you. I wasn't trying to make you answer all that shit.

Sorry about that!

All the revisions/additions looked excellent to me!

(Pixie's gonna LOVE this game!)

**Comment by KVonGit** [comment]
Updated the game. 

(Made my bits into 2 libraries, so be sure to clone the whole directory (or download as a ZIP and unzip (or get both libraries along with the main game (they are kvcommandsLib.aslx and kvLib4px.aslx))).

...or I can put it back into the game file, if everyone prefers...

**Comment by KVonGit** [comment]
FIXED:

**buttonXM:**

can press or push

**instrumentsXM:**

'play instruments' runs the 'play drums' script

**Comment by XanMag** [comment]
I assume nothing will explode if I've already fixed it and I upload my new code?

**Comment by KVonGit** [comment]
> I assume nothing will explode if I've already fixed it and I upload my new code?
> …

Go ahead and upload, good sir!

**Comment by KVonGit** [comment]
I took the liberty of converting your drums only track from the 15mb .wav to a 1.5mb .mp3, XM.

(They call me The Space-Saver. Mm-hmm.)

**Comment by XanMag** [comment]
That'll work.

**Comment by KVonGit** [comment]
Just posted an update.

If anyone has edited recently, it's cool. 

Just post a comment here whenever you commit. (I get an email for everything EXCEPT for the most important thing!)

If we cross the streams, I can fix it. (I've got GNU/Linux powers, bitches! Yay-yuh!)

**Comment by KVonGit** [comment]
Just updated the shit out of it and checked it back in.

Check it out, you guys!

**Comment by KVonGit** [comment]
Got word from Alex Warren, too.

He now provides light in a dark place.


---
Is DL's clone going to steal the shard Pixie finds in Crate Mart at the end of the game?

Will XM have beer and daydreams (and fantasy football)?

And what the hell's on that WORD DOCUMENT???


The answers to these questions, and many more, on the next episode of:

# SHARDS OF &infin;

**Comment by KVonGit** [comment]
PS

I have things all set up to fuck with Pix, but you can breeze through all my stuff right now.

Just to warn everyone, though:

Crates will fly in the Crate Mart by Monday.

---
### Another Note

I took most of my stuff out of the main file and put it into kvLibPixiesQuest.aslx.

Just to make everyone's edits go more smoothly.

So, be sure to clone the repo and work from the new folder! Or at least download the library, pictures, audio....

Yeah... Just download everything. I deleted most of the extra files, too... so it's smaller now.

**Comment by KVonGit** [comment]
Just updated quite a few things in the main area.

**Comment by DarkLizerd** [comment]
I think we need a revision number attached to the 4Pixie.aslx file. I think we should be up to V. 0.0.9 or so.
My changes were dropped from the most recent update.
And, as for the external files, kvLibPixiesQuest.aslx and InvPane2.aslx, there should not be any reason that these should be external files. 
Cool new interface tho...

**Comment by KVonGit** [comment]
>And, as for the external files, kvLibPixiesQuest.aslx and InvPane2.aslx, there should not be any reason that these should be external files.

They aren't.

>I think we need a revision number attached to the 4Pixie.aslx file. I think we should be up to V. 0.0.9 or so.
 There is. I even put the time and date on it.

>My changes were dropped from the most recent update.

What? I'll investigate that right now! I'm sorry!

XM just submitted an update. I merged the files by hand, and I must have messed something up!

>Cool new interface tho...

Thank ya, thank ya.

I'm getting rid of all those damned files in the folder.

I fell asleep doing it at 4 this morning.


---
Going to fix you part right now, DL! I'm sorry!

**Comment by KVonGit** [comment]
```xml
  <game name="Pixie's Quest">
    <gameid>7e2cf517-9039-4061-bf0d-8c3540e30372</gameid>
    <version>0.0.21</version>
    <firstpublished>2017 08/27/2017 1amCST</firstpublished>
```


What does yours say right here, DL?

**Comment by KVonGit** [comment]
\> go to DL's Room
You step through the doorway and feel like you just stepped onto a descending elevator . You stumble a few steps before catching your balance, but you still feel the elevator effect.

**DL's Room**
You are in a large field covered by an off-green grass, and a pale orange sun hangs in the cloudles sky over head. Before you, is a desk with scattered papers all around. Behind the desk is a person working hard working some sort of problem. You watch as he picks up one paper, looks at another, then scribbles a note on both before setting them to the side. You watch as one of the papers slide off the edge of the desk and float softly to the ground.
In the corners of the room are trees casting dark shadows underneath. Every now and then, you think you see something move!


You can see DarkLizerd and a desk.
You can go west, north, east or south.


---
Is this the old prose, DL?

(I'm currently digging through the old versions, but this is what's in the version in the master repo right now.)

**Comment by KVonGit** [comment]
\> help DarkLizerd
You ask if there is any way you could help with the work he is working on. He replies: I tried to create a clone of me to help with this work but it did not turn out so well and I had to lock it away. From one of the dark trees you hear a voice call out, "I am not the clone you stupid lizard! I am!" He yells back, "NO YOU ARE NOT!!! Would a clone know what happened before he was cloned???!!! I think not! Therefore I am the original and not YOU!" From another tree, you hear, "IDIOT!!! When you were cloned from ME, you learned everything I knew, INCLUDING WHAT HAPPENED BEFORE!!!" The one at the desk yells back, "Yea??? What about this scar across my forhead! Why would that get cloned?" (You suspect there is nothing to be gained by listining to any more of this.)


---
This is the newest stuff I can find, and it's in the current version of 4px.aslx.

The latest addition from you I can find is from August 20th.

---

**Comment by KVonGit** [comment]
DL,

The easiest way on you would probably be to just email me the latest 4px.aslx you have, and I'll fix it all.

I'm sorry, man!

---
That's why I had a separate library for my stuff for a minute. 

When 2 of us update at once, each complete game's code has to be compared against the other...

That's pretty daunting, and that's why Pixie went directly to the library system for the group project. (He didn't want be the asshole (me) who accidentally cut someone's parts out. (Along with all sorts of other reasons...))

Which reminds me: That Pixie: He's one bad mofo!

**Comment by KVonGit** [comment]
NOTE:

I just read over this, and it could easily be construed as dicky.

You should imagine the voice of this guy: He's had about 4 beers and a few shots of J.W...

He's not dicky. (He's just an ass-half. (It takes two of him to make an ass whole.))

**Comment by KVonGit** [comment]
Here are the new files needed to properly play (and edit).

NOTE: You don't REALLY need the .txt files, but they're small, so... I left them because I think it's cool being able to read a file from the local directory from inside the game. If you don't get them in your folder, a few little commands won't work, but it won't effect the game as a whole.



https://github.com/KVonGit/HVACR-specifications/tree/master/refrigerant%20charts

I've got it down to 13 files now.

No more libraries at all.

---
I can convert mp3 files to Base64, and they play perfectly in a normal HTML, but I can't pull it off in Quest as of yet.

The images work fine though, with the exception of the two background images I use. I can't get that to work as Base64 code, so there are still two PNG files in the folder. (One is the cover image, though. The only image in the folder that's really 'extra' is script.png, and I use it for the background image for one turn when you examine the glitch.)

---
NOTE: The images disappeared from the folder, as did the libraries...

...just don't curse me while in Code View. Okay, guys?


---
I'll be updating again shortly.

I'm about to take a lot of the Quest commands out.

---
More on this as it comes in.

**Comment by DarkLizerd** [comment]
The last version I worked on was 0.0.8, That one had my updates, mainly just getting the button on the planet disk working, and a few wording corrections (changes).
8-20 would be about the last time I worked on it.
What I downloaded yesterday is ver 0.0.21...
And that is where I noticed my changes missing.
(This was the reason for the  check out list, so 2 people would not override each other's updates, and having the version # in the file name to make sure that everyone is working from the latest copy.
I think I've got my changes in my copy of .21... provided no one is working on their part.

**Comment by KVonGit** [comment]
Go ahead and upload it, DL.

I just checked it back in.

**Comment by KVonGit** [comment]
>What I downloaded yesterday is ver 0.0.21...
And that is where I noticed my changes missing.

I think that was in between the file merging.

What had happened was:

XM downloaded what was online, which didn't include your revisions because I was adding some things onto the same file before uploading.

XM and I committed revisions nearly simultaneously, so I decided to just edit it by hand.

I just plain messed it up after that, but you should see the DIFFs now! It's insane!

...anyway, later that night I ran a DIFF on what I had before I merged XM's stuff in. (I always be saving a backup first; know what I'm sprayin'?)

Then... I added your stuff and some of my stuff I missed back in and committed it again.

---
# This is when I pulled my bits out and made libraries.

I've been behaving myself and checking it in and out changing the version number and stuff.

It still gets out of whack.

It's my fault, though. I'm just sitting here updating the mechanics and changing the display settings.

Like right now, it's version 24, but it was version 22 just this morning.

Version 22 had a shitload of dependencies (the image and document files and libraries), but version 24 barely has any, and the display settings are totally different.

Now, anyone in their right mind (that's you DL and XM) is probably revising their respective bit of version 21. So, whenever those updates are submitted, it will delete everything added since version 21, and I'll be adding them back in manually.

I'm not complaining at all, mind you. Just pointing out that the library system allows everyone to be creative as the mood strikes them without waiting or re-revising or worrying about messing up someone else's parts of the game.

The library system is a pain in the ass, too, and I'm not saying we should switch to it.

Maybe we should turn on the thing where no one can update within 24 hours of the last update...

..nah. I don't think that would do anything worthwhile upon reflection.

---
F--k it!

We've got it going on, guys!

I've got a backup of every single thing anyone has submitted.

To hell with the libraries!

If shit goes astray, I've got our backs! (As long as everyone points it out when I mess up!)

---
Drop a comment here when you check it in or out, please, everyone.

... just so it sends out a notification.

**Comment by DarkLizerd** [comment]
Don't beat yourself up over this... 
(Well do it to you later... 😈 )
I got v.24... checking now...

**Comment by KVonGit** [comment]
I definitely messed the button up.

I've got it all right here though, DL.

https://raw.githubusercontent.com/KVonGit/HVACR-specifications/b51c8fdbf06f42e79dd4bb5df5920d5a30c487b1/refrigerant%20charts/4px%200.0.8.aslx

It got messed up again after that time I was commenting on each thing for archival purposes.

And see? Now I can just use those! Damn! I was thinking for a second there!

**Comment by KVonGit** [comment]
DL,

I can drop this all back in the game right now and commit so you won't have to do all that re-writing...

```xml
<object name="DL_Room">
    <inherit name="editor_room" />
    <description><![CDATA[You are in a large field covered by an off-green grass, and a pale orange sun hangs in the cloudles sky over head. Before you, is a desk with scattered {object:papers} all around. Behind the desk is a person working hard working some sort of problem.  You watch as he picks up one paper, looks at another, then scribbles a note on both before setting them to the side. You watch as one of the papers slide off the edge of the desk and float softly to the ground.<br/>In the corners of the room are {object:trees} casting dark shadows underneath. Every now and then, you think you see something move!<br/><br/>]]></description>
    <usedefaultprefix type="boolean">false</usedefaultprefix>
    <attr name="implementation_notes"><![CDATA[This world is not on this planet...<br/><br/>Side note... Extra Dark Lizerds hiding in other worlds could be "fun"...<br/>(but not lizards that are "extra dark"...)<br/>]]></attr>
    <alias>DL's Room</alias>
    <beforefirstenter type="script">
      MoveObject (kv_voicemail, kv_device)
    </beforefirstenter>
    <beforeenter type="script">
      msg ("You step through the doorway and feel like you just stepped onto a descending elevator . You stumble a few steps before catching your balance, but you still feel the elevator effect.")
    </beforeenter>
    <object name="DarkLizerd">
      <inherit name="editor_object" />
      <inherit name="namedmale" />
      <alt type="stringlist">
        <value>DL</value>
        <value>dark</value>
        <value>lizard</value>
        <value>lizerd</value>
      </alt>
      <listalias>Dark Lizerd</listalias>
      <takemsg>Sorry, but I'm not to be taken for granit. </takemsg>
      <inroomdescription type="string"></inroomdescription>
      <take type="boolean">false</take>
      <usestandardverblist type="boolean">false</usestandardverblist>
      <useindividualverblist type="boolean">false</useindividualverblist>
      <look><![CDATA[You see a person wearing a dark grey or black cowl that hides most of his features. For some strange reason, you are not totaly comfortable around him. You almost get the fealing he is not quite human.<br/>]]></look>
      <displayverbs type="stringlist">
        <value>Look at</value>
      </displayverbs>
      <project>I'm helping with a secret project for a friend as a way of saying 'Thank you'... But it's a secret so you can't tell anyone, OK?</project>
      <papers>I'm trying to translate Basic to Quest, so that I can submit all the Basic projects I've been working on and a few more that I can translate. But the problem is that 'print' is now 'msg', and 'select case' is now 'switch' and 'switch' is also for turning things on and off, and 'input' is now 'GetInput' but there is no varable to collect the answer from, and Quest can convert Aribic to Roman, but only the numbers, and Spaces(5) will print 5 spaces, (or will it\" msg\" 5 spaces?) and there are objects that can be objects, or not objects, then there is no way to save data to a text file, but there is a way to read that file, but you cannot tell Quest how you want to read it, then you gota figure out what you read which is hidden in some variable that is preprogrammed into Quest.....  (5 minutes later, altho it felt like hours)... you back away.</papers>
      <help>You ask if there is any way you could help with the work he is working on. He replies: I tried to create a clone of me to help with this work but it did not turn out so well and I had to lock it away.  From one of the dark trees you hear a voice call out, "I am not the clone you stupid lizard! I am!" He yells back, "NO YOU ARE NOT!!! Would a clone know what happened before he was cloned???!!! I think not! Therefore I am the original and not YOU!" From another tree, you hear, "IDIOT!!! When you were cloned from ME, you learned everything I knew, INCLUDING WHAT HAPPENED BEFORE!!!" The one at the desk yells back, "Yea??? What about this scar across my forhead! Why would that get cloned?" (You suspect there is nothing to be gained by listining to any more of this.)</help>
      <askdefault type="script"><![CDATA[
        msg ("Can't you see I'm just a little busy translating Quest to Basic... No, that's not right, I'm translating something to Quest Basic, or was it to Basic Quest??? KILETA FRO KIM TAT!!! Now I confused!! Thanks!!! Now I gota start over!!! <br/>(You back up wondering just how far the tantrum will extend...)<br/>")
      ]]></askdefault>
      <telldefault type="script"><![CDATA[
        msg ("Can't you see I'm just a little busy translating Quest to Basic... No, that's not right, I'm translating something to Quest Basic, or was it to Basic Quest??? KILETA FRO KIM TAT!!! Now I confused!! Thanks!!! Now I gota start over!!! <br/>(You back up wondering just how far the tantrum will extend...)")
      ]]></telldefault>
      <ask type="scriptdictionary">
        <item key="project">
          msg ("I'm helping with a secrect project for a friend as a way of saying 'Thank you'... But it's a secrect so you can't tell anyone, OK?")
        </item>
        <item key="papers"><![CDATA[
          msg ("I'm trying to translate Basic to Quest, so that I can submit all the Basic projects I've been working on and a few more that I can translate. But the problem is that 'print' is now 'msg', and 'select case' is now 'switch' and 'switch' is also for turning things on and off, and 'input' is now 'GetInput' bur there is no varable to collect that info from, and Quest can convert Arabic to Roman, but only the numbers, and Spaces(5) will print 5 spaces, (or will it msg 5 spaces?) and there are objects that can be objects, or not objects, then there is no way to save data to a text file, but there is a way to read that file, but you cannot tell Quest how you want to read it, then you gotta figure out what you read, which is hidden in some variable that is pre-programmed into Quest..... <br/>(You back away while you still have your sanity...)<br/>")
        ]]></item>
        <item key="help"><![CDATA[
          msg ("You ask if there is any way you could help with the work he is working on. He replies: I tried to create a clone of me to help with this work but it did not turn out so well and I had to lock it away.  From one of the dark trees you hear a voice call out, \"I am not the clone you stupid lizard! I am!\" He yells back, \"NO YOU ARE NOT!!! Would a clone know what happened before he was cloned???!!! I think not! Therefore I am the original and not YOU!\" From another tree, you hear, \"IDIOT!!! When you were cloned from ME, you learned everything I knew, INCLUDING WHAT HAPPENED BEFORE!!!\" <br/>(You back up not knowing if you will be included next...)<br/>")
        ]]></item>
      </ask>
    </object>
    <exit alias="west" to="Quest Forum">
      <inherit name="westdirection" />
      <message type="string"></message>
      <runscript />
      <script type="script">
        msg ("You stumble out through the door and almost fall down. You feel like a heavy weight is pressing down on you. (strange!)")
        MakeObjectVisible (Dark_Lizerd)
        MoveObject (player, Quest Forum)
      </script>
    </exit>
    <exit name="DLN" alias="north" to="DL_Room">
      <inherit name="northdirection" />
      <lookonly type="boolean">false</lookonly>
      <look>You start to wander to the north to see just how far the field extends when you suddently run smack into a clear wall. (You hear a snickering behind you) You examine the wall but discover that the view beyond is not painted on the wall, but is instead beyond it.</look>
      <lockmessage>You start to wander to the north to see just how far the field extends when you suddently run smack into a clear wall. (You hear a snickering behind you) You examine the wall but discover that the view beyond is not painted on the wall, but is instead beyond it.</lockmessage>
      <locked />
    </exit>
    <exit name="DLE" alias="east" to="DL_Room">
      <inherit name="eastdirection" />
      <lookonly type="boolean">false</lookonly>
      <look>You start to wander to the north to see just how far the field extends when you suddently run smack into a clear wall. (You hear a snickering behind you) You examine the wall but discover that the view beyond is not painted on the wall, but is instead beyond it.</look>
      <locked />
      <lockmessage>You start to wander to the east to see just how far the field extends when you suddently run smack into a clear wall. (You hear a snickering behind you) You examine the wall but discover that the view beyond is not painted on the wall, but is instead beyond it.</lockmessage>
    </exit>
    <exit name="DLS" alias="south" to="DL_Room">
      <inherit name="southdirection" />
      <lookonly type="boolean">false</lookonly>
      <look>You start to wander to the north to see just how far the field extends when you suddently run smack into a clear wall. (You hear a snickering behind you) You examine the wall but discover that the view beyond is not painted on the wall, but is instead beyond it.</look>
      <locked />
      <lockmessage>You start to wander to the south to see just how far the field extends when you suddenly run smack into a clear wall. (You hear a snickering behind you) You examine the wall but discover that the view beyond is not painted on the wall, but is instead beyond it.</lockmessage>
    </exit>
    <object name="trees">
      <inherit name="editor_object" />
      <inherit name="plural" />
      <scenery />
      <look>You start to examine the trees, but something about them makes you nervous so you back away.</look>
      <usestandardverblist type="boolean">false</usestandardverblist>
      <displayverbs type="stringlist">
        <value>Look at</value>
      </displayverbs>
      <inventoryverbs type="stringlist" />
    </object>
    <object name="papers">
      <inherit name="editor_object" />
      <inherit name="plural" />
      <inroomdescription type="string"></inroomdescription>
      <take type="boolean">false</take>
      <takemsg>You reach for the paper, but a long green arm snatches it first. 'Thanks, I was looking for that one.' he says.</takemsg>
      <look>The scattered papers have all sorts of writing and scribbles with boxes and arrows and arrows pointing to other boxes with circles pointing to other shapes... For a moment you almost understood what you were looking at... Almost.</look>
      <scenery />
      <ontake type="script">
      </ontake>
    </object>
    <object name="desk">
      <inherit name="editor_object" />
      <inherit name="surface" />
      <look><![CDATA[The desk is made out of a dark wood, oak perhaps. On top is several piles of {object:papers} with strange writings and scribbles everwhere. {if disk.taken=False:{if globe.visible: Over to the side is a floating, spinning {object:globe} of a planet you don't reconize. He watches you for a moment before speaking, 'That is a globe of my home world, where we are now. If you like it, take it, I got more around here."} {if disk.visible: There is a {object:disk} about the size of a hockey puck sitting here.}}<br/>]]></look>
      <displayverbs type="stringlist">
        <value>Look at</value>
      </displayverbs>
      <feature_container />
      <hidechildren />
      <listchildren type="boolean">false</listchildren>
      <inventoryverbs type="stringlist">
        <value>Look at</value>
        <value>Drop</value>
      </inventoryverbs>
      <object name="globe">
        <inherit name="editor_object" />
        <takemsg>You reach for the globe, but your hand passes right through it. Altho you could not get it, it felt like you hand passed through a bubble of water, but you hand is still dry!</takemsg>
        <scenery />
        <usedefaultprefix type="boolean">false</usedefaultprefix>
        <look>It is a floating, spinning globe of a planet you don't reconize. There are strange land masses with large and small green tinted blue oceans. </look>
        <inventoryverbs type="stringlist">
          <value>Look at</value>
          <value>Drop</value>
        </inventoryverbs>
        <take type="script">
          msg ("You reach for the globe, but your hand passes right through it. Altho you could not get it, it felt like your hand passed through a bubble of water, but your hand is still dry!")
          msg ("\"Here.\" He says and reaches out and touches a {object:disk:disk} that you didn't notice under the globe, and the planet disappears.")
          MakeObjectInvisible (globe)
          MakeObjectVisible (disk)
        </take>
      </object>
      <object name="disk">
        <inherit name="editor_object" />
        <take />
        <visible type="boolean">false</visible>
        <usedefaultprefix type="boolean">false</usedefaultprefix>
        <look>It is a disk about the same size as a hockey puck, in fact, you would have thought it was one except for the small raised bump on one face.</look>
        <feature_container />
        <scenery />
        <alt type="stringlist" />
        <alias>planet disk</alias>
        <feature_usegive />
        <inventoryverbs type="stringlist">
          <value>Look at</value>
          <value>Drop</value>
        </inventoryverbs>
        <givesingle type="boolean">false</givesingle>
        <nocopy type="string"></nocopy>
        <pushbutton type="script">
          if (not ListContains(ScopeVisible(), globe)) {
            MakeObjectVisible (globe)
            msg ("The planet re-appears, and starts to rotate slowly.")
          }
          else {
            MakeObjectInvisible (globe)
            msg ("The planet disappears from view.")
          }
        </pushbutton>
        <giveto type="scriptdictionary">
          <item key="Richard Headkid"><![CDATA[
            RemoveObject (Richard Headkid)
            msg ("As soon as you hand the disk over to RH, he produces a Strange Pouch, reaches one hand into it, and disappears!<br/><br/>(You now have access to his desk.)<br/>")
            MakeObjectVisible (RH desk)
            MoveObject (disk, Richard Headkid)
            myscript => {
              msg ("\"He went {command: go north: north},\" says Kavin.")
            }
            dictionary add (KV.ask, "rh richard headkid", myscript)
            SetObjectFlagOn (disk, "been_taken_by_rh")
            list remove (disk.generatedverbslist, "Give to RH")
          ]]></item>
        </giveto>
        <ontake type="script">
          SetObjectFlagOn (disk, "taken")
        </ontake>
        <taken type="boolean">false</taken>
        <object name="planetdiskbutton">
          <inherit name="editor_object" />
          <look>A small black button that almost blends into the surface.</look>
          <take />
          <takemsg>How do you expect to take the button off the disk?</takemsg>
          <alias>button</alias>
          <scenery />
          <alt type="stringlist">
            <value>button</value>
            <value>bump</value>
          </alt>
        </object>
      </object>
    </object>
  </object>
  <verb name="dl_project">
    <property>project</property>
    <pattern>project</pattern>
    <defaultexpression>"You can't project " + object.article + "."</defaultexpression>
  </verb>
  <verb name="dl_papers">
    <property>papers</property>
    <pattern>papers</pattern>
    <defaultexpression>"You can't papers " + object.article + "."</defaultexpression>
  </verb>
  <verb name="dl_help">
    <property>help</property>
    <pattern>help</pattern>
    <defaultexpression>"You can't help " + object.article + "."</defaultexpression>
  </verb>
  <verb name="dl_otherone">
    <property>otherone</property>
    <pattern>other one</pattern>
    <defaultexpression>"You can't other one " + object.article + "."</defaultexpression>
  </verb>
  <verb name="dl_clones">
    <property>clones</property>
    <pattern>clones</pattern>
    <defaultexpression>"You can't clones " + object.article + "."</defaultexpression>
  </verb>
```

**Comment by DarkLizerd** [comment]
OK, that's the changes I made, but there were a few other tweaks in what I had locally.
I copy and paste from .8 to .24 and add back in what I had here...
(but thanks anyway...)

**Comment by DarkLizerd** [comment]
You want Wumpus???
I got a few ideas...
1) Wumpus 3.0 with room descriptions... (as is)
2) Wumpus 3.0 edited for a preset settings...
3) Wumpus 3a new game world set for this game.
If #3, then I need a trigger event when everyone is dancing that opens a new exit in my room...
(The harmonics of the music causes a "door" to open...)

**Comment by KVonGit** [comment]
Whichever way you feel's best, DL.

---
Does the music piss off the Wumpus?

---
I put that you walk the dinosaur during the big dance number, but that's because that was the only dance DarkLizerd brought to mind, by the way.

What would DarkLizerd do during the dance number?

**Comment by DarkLizerd** [comment]
More like it just opens the door... then the music wakes it up...
I guess it causes it to go on a rampage... Then the idea would be to find a way to close the door, in the Wumpus realm... 
Sounds like option 3...
So... I'll update V.24 and use this week to write the Wumpus section...

**Comment by KVonGit** [comment]
After you upload your recent revisions, I'm gonna change it back to where my part is a library.

That way, I won't go crazy sitting here having all sorts of ideas about things to add and change.

I'll leave everyone else's stuff in the main game file, though.

I know I keep going back and forth on the library, but I've thought up and forgotten at least 10 things to change in the game over that past hour. We can all edit when we please with if I put my part in a library.


So, here's what I propose:

Upload what you've got once you add in the stuff you've got now and submit it.

I'll change it so my part is a library and upload the main game right back for anyone to do whatever whenever.

That sound cool?

**Comment by DarkLizerd** [comment]
Actually... If your section is just in a library, then it would not matter who has the file, you just update your section when the file id free...
Or, write your part for the main file, but all your changes are under your "code block", just like my DL_Room". Then, when you get the main code, delete your section, and drop in your re-write, then any tweaks for the main hub, and re-upload it...

**Comment by KVonGit** [comment]
Yeah, I know. That's what I'm going to change it to.

Right now I have things scattered everywhere.

I'll be taking all my stuff back out of the main room and moving it in on Initialization.

I had it like that before, but I realized everyone would have to put my libraries in their folders before they could even open the game in the editor, so I changed it all back.

I'm fooling with the commands and the display settings an the phone and the book that the player object is carrying, so it's not just my rooms I'm editing.

**Comment by KVonGit** [comment]
I've made it complicated. That's what's the deal really is.

I'm fixing it as soon as it's checked back in though, and it'll be smooth sailing for everyone from there.

**Comment by DarkLizerd** [comment]
Help... I broke my doorway...
I want it to be an object in the hub room description, and going east will no be a problem.
I've also added a keypad that can be used to enter some test cheat codes for what I'm planning on adding to my room... Like a helper and eating, based on a random number.

**Comment by KVonGit** [comment]
Email me your aslx file.

We'll get that damned door fixed up in a jiffy!!!

**Comment by KVonGit** [comment]
...or you can just commit it to the master repo...

That or email... Same difference.

**Comment by DarkLizerd** [comment]
If you can just post the change here... thanks...

**Comment by KVonGit** [comment]
I gotcha.

I don't see the error in Notepad++, but Quest won't open it in the editor.

I'm close though. I can smell it...

**Comment by DarkLizerd** [comment]
Quest loads it for me!...
You can go to strange doorway.
> go strange doorway
I can't see that.
but:
>e
works... And it is not visible in the compass.

**Comment by KVonGit** [comment]
I re-downloaded and the file opened right up.

There are Gremlins in my internet sometimes...

**Comment by KVonGit** [comment]
![image](https://user-images.githubusercontent.com/30656341/29808463-cca0effc-8c5d-11e7-919d-5f2cfe9c56ec.png)

Okay... it isn't scenery...

**Comment by KVonGit** [comment]
![image](https://user-images.githubusercontent.com/30656341/29808560-41e3ccbc-8c5e-11e7-8b37-7e1d246167fb.png)

![image](https://user-images.githubusercontent.com/30656341/29808573-50693a1a-8c5e-11e7-9c73-3c85b4ce95c1.png)

**Comment by KVonGit** [comment]
That was it.

Change it to non-directional exit.

**Comment by DarkLizerd** [comment]
That is too simple... there's got to be something more!!!
(Hides in corner in shame...)

**Comment by KVonGit** [comment]
>That is too simple... there's got to be something more!!!
(Hides in corner in shame...)

Pshht!

I did this last night:

```xml
<object name="kv_fucked_this_up" />
  ```

That fucking forward slash kicked my ASS for an hour!!!

...usually, ```<![CDATA[``` is what gets me...

**Comment by DarkLizerd** [comment]
So... why did it not work as a directional exit???

**Comment by KVonGit** [comment]
> If you can just post the change here... thanks...

I've been hanging out with a bottle of Mr. Walker, so I'm a tad slow to interpret English right now....

You want me to just commit the whole file (with the fix) to the repo?

Or just comment what I found right here?

Either way is just a click... Just checking which way you likey.

**Comment by KVonGit** [comment]
>So... why did it not work as a directional exit???

If you want EAST and GO TO STRANGE DOORWAY, make it a normal EAST, then change the alias to STRANGE DOORWAY.

I ***think*** that will make it work either way...


It did. It took our ass east, didn't it?

**Comment by DarkLizerd** [comment]
Anyone reading this will (hopefully) see the edit, but, if you got fast internet, go ahead an re-upload it...
When I made it east, it broke...
I was starting to not like the "DL_Room" label and felt that "strange doorway" would be more keeping with a story...

**Comment by KVonGit** [comment]
Yeah, I just broke mine too...

How do you get in?

Nevermind, don't tell me...

Give me two minutes.

**Comment by KVonGit** [comment]
Well, I made GO TO STRANGE DOORWAY work, but I can't figure out the code to see if I can get in.

**Comment by KVonGit** [comment]
Nevermind. Just had to try to go to door twice...

Duh!

**Comment by DarkLizerd** [comment]
I guess you didn't check out the keypad?

**Comment by KVonGit** [comment]
Naw...

I examined the keypad, but I just tried to use the card.

Should have examined that card...

**Comment by KVonGit** [comment]
I just committed to the main file.

4px.aslx

I'm about to remove the one labeled 0.0.25 now. (I have it downloaded for archival purposes.)

**Comment by KVonGit** [comment]
Nice additions, by the way, sir!

...and you do the Wumpus the way it feels right to you! The Wumpus is your bitch! You're Charlie! This is your chocolate factory!

**Comment by KVonGit** [comment]
Just pushed a commit to the master repo.

Argh! Now I'm speaking Git!

I just updated the file.

Checked back in!

**Comment by KVonGit** [comment]
# Just updated the main game.

## I added Quests!

[How to ADD and COMPLETE Quests](https://github.com/KVonGit/HVACR-specifications/issues/23)

---
Whenever anyone is getting ready to edit or add anything, just let me know!

I'll check it right back in and submit what I'm working with so you can work from the most current file.

**Comment by KVonGit** [comment]
I just committed version 0.0.30.

I'm checking it back in (at least for 5 minutes), and going to play with kvLib.

If kvLib is not included: the main game has an alternate RH and he'll grant you access to a very short run through his world. I also have it set so it won't throw any errors if kvLib is not included.

NOTE: If kvLib is not included, the game title will have an asterisk displayed after it.

#### Pixie's Quest *


---
#### JUST FOR FUN:

If you want to look at some crazy coding, check out the ASK ABOUT script on DumbRH. (His world is created totally on the fly if you answer YES when he invites you to visit it. (Well... I can't figure out how to create a command on the fly, so there's a blank command in the beginning then I alter the pattern when the player examines the token for the first time. But, besides the command, everything is created on the fly upon entering RH' World. Only DumbRH and the command exist before you enter RH's World.))

---
[top](#top)

**Comment by KVonGit** [comment]
Damn it!

Had stuff messed up!

Just committed...

version 0.0.32

**Comment by KVonGit** [comment]
Damn it again!

Had stuff messed up again!

Just committed again...

version 0.0.33

---

Really checking it back in and working on something else for a bit now!

---

Pixie's back!

How much more stuff is everyone planning to add?

I am adding a puzzle in Crate Mart, but that's all I've got planned.

Might take me one day. Maybe two. (Depending on whether or not I can keep my happy ass off of the forum long enough to write some game!)

Not rushing. Just wondering...

**Comment by XanMag** [comment]
I've got a couple days at least.  I have 2-3 rooms to add with a puzzle in each.  I think the most complex two are done.  Each room might be 90 minutes of work, but the problem is finding 90 minutes to get it done.

**Comment by DarkLizerd** [comment]
I have a question, and this will determine how I do the Wumpus part...
What is it about Wumpus that you guys like?
I got 2 reviews, 1 1 star, 1 5 star, 1736 plays, and 138 downloads...
(as of today)

**Comment by KVonGit** [comment]
>What is it about Wumpus that you guys like?

I've tried to find that damned Wumpus for at least 5 minutes every day for a couple of months!

I just want to shoot that elusive bastard!


NOTE:

I finally broke down and pushed button 0.

That's some funny stuff, right there!

NEVER PUSH BUTTON 0!

**Comment by DarkLizerd** [comment]
Button "0" was an "on the fly" addition...
So... it's the hunt?
the random maps?
the little, easy to miss, clues?

**Comment by KVonGit** [comment]
I don't know...

The randomness is pretty cool...

...but I think it's just the idea of the Wumpus.

...sort of like a Grue...

---
# UPDATE


>\> go to Door 7
You shoot and kill the Wumpus! You win!


---
Ha! 

I finally got that bastard!!!

---
Seriously, DL,

The stuff you've written for this is already way more awesome than the Wumpus stuff.

Go with what you feel.

I thought a Wumpus part, or just a reference, would be cool, but don't force it, man! Let the art flow!

**Comment by DarkLizerd** [comment]
Sounds like you get version 1 or 2...
Verson 2.5 has named exits.
V3.0 has random room descriptions...
And I'm thinking of changing the arrows to "gas bombs"  that only knock out the Wumpus, then, you need to take some blood, and carry it in something, back to room 1 where it will open the door...
Just don't loose the blood or you will need to do this all over again...
(And the Wumpus will have moved because you disturbed it...)
I did had an idea of the Wumpus hunting the intruder... (you)...

**Comment by DarkLizerd** [comment]
Actually... You do have V.2.5...
I was also thinking of a "torment" difficulty level... No rooms are named, you get to name them, the exits go to in-named rooms...
After all, you are the first one to explore the cave and no one has left any marks for you to follow... AND, there is no reason to expect the door you just came through connects back to the room you just left...
Now, you will really get lost...

**Comment by DarkLizerd** [comment]
... Be afraid... Be VERY afraid!!!
You are now in the Wumpus's menu...

**Comment by KVonGit** [comment]
Just checked it back in.

Version 0.0.39


kvLib is not separate and is included in the game at the moment.

I just changed/added/altered quite a few things, and I'm tired of Questing (for at least an hour).

**Comment by KVonGit** [comment]
Okay, it's really checked back in again!

---
TODO:
- [ ] Finish setting up the hints thing (which I haven't showed you guys yet, but it's simple and cool).
- [ ] Something else that I can't recall.

**Comment by KVonGit** [comment]
Checked it back out.

Playing with the hints and completed Quests thingy.

**Comment by KVonGit** [comment]
Just drop a comment if you'd like to edit your bit.

I'll check it back in.

(I'm just polishing things up.)

**Comment by KVonGit** [comment]
Just checked it back in.

I'll quit fiddling with things, and let you guys have access to it.

Sorry about that.

Always feel free to yell at me that it's someone else's turn if you ever feel the urge!

**Comment by KVonGit** [comment]
Had the Telemat booth messed up. 

Just uploaded version 0.0.48.

**Comment by KVonGit** [comment]
Checking it out for a few minor fixes.

If our commits overlap, I just learned how to use ```git mergetool```, so we're cool.

**Comment by KVonGit** [comment]
I just checked it back in.

kvLib.aslx is in the folder but it's NOT included by default!  The RH that's there is just a placeholder when he's mumbling about kvLib.aslx.

---

Is anyone planning on using the money feature?

KV
- [ ] yes
- [x] no

DL
- [ ] yes 
- [ ] no

XM

- [ ] yes
- [ ] no

**Comment by DarkLizerd** [comment]
No, but leave it in, there could be a use later... depending on how big this is going to grow...

**Comment by KVonGit** [comment]
Cool.

I'm turning it off in the status pane when play begins.  If the player ever acquires any money, "Money" will appear in the status pane from there on out.

---

Start Script:
```C++
dictionary remove(game.povstatusattributes, "money")
```

Turn script (checkForMoney):
```C++
if (player.money > 0) {
  dictionary add (game.povstatusattributes, "money", "Money: !")
}
DisableTurnScript (checkForMoney)
```

**Comment by KVonGit** [comment]
Just checked the game back in.

**Comment by KVonGit** [comment]
Checking it out for a quick fix.

**Comment by KVonGit** [comment]
Okay.  Checked back in.

**Comment by KVonGit** [comment]
I'm checking it out, hitting three buttons, and checking it back in.

**Comment by KVonGit** [comment]
Checking it out.

Just yell at me if you get an urge to edit.

**Comment by KVonGit** [comment]
Checked back in.

0.0.66

Be sure to download kvLib too!  It is included as a library at the moment.

**Comment by KVonGit** [comment]
Damn it!

I've got a pane out of order!

Checking back out for one minute.

**Comment by KVonGit** [comment]
Okay...

It's checked in.

I'm going to go finish porting They Call Ya' Mister now, to give you guys some space to work.

---
Recently added:
- Player is now carrying Doctor Agon's Text Scrambler
- Added Quests and functions
- Added maxScore
- Added Pixie's book of magic
- Added stats in location bar
- Added hints and ```addHint``` function
- Probably something else, too...
- Oh yeah, I made a function to verify the player's Pixie-ness, but I didn't activate it yet.  (It will mess up all the walk-throughs, which can be edited, but not right yet. (Plus, I don't feel like logging in as Pixie every time I boot the game up!))

---
**TODO:**

After you guys declare that your parts are finished, we'll come up with some kind of ending.  (Maybe DL's clone grabs the shard Pixie finds in Crate Mart and runs off, leaving us with nice cliffhanger...)

**Comment by KVonGit** [comment]
PS

When the 2-step verification process ***is*** set up, this will be the Boolean:

```C++
game.isPx = true
```

**Comment by KVonGit** [comment]
Aargh!

I added a command to use Pixie's Caesar Cipher script too!

```C++
cipher ("ABCD", 2)
decipher ("BCDE", 2)
```


---
I fixed all the enter/go scripts, too.


---
Hrmm.... Still forgetting something...

I added a real-game-time timer to the game, but I haven't implemented it yet.

I will probably have Pixie have to solve the puzzle in Crate Mart in a limited amount of time or something...


---
Oh!

I packaged most of the crazy commands I had in there into scrolls Pix will have to collect throughout my parts.  He doesn't get the CHEAT commands until Crate Mart now, and I set it where he can't view an .aslx file from within the game.  (He CAN view any text documents in the main game's folder though, if you guys want to play off of that.)

AND

The Quest PX has been UPDATED LIKE A MOFO!!!  USE it to check it out!


...and if you want to 'send' Pixie a notification or post link at any point during play, let me know and we'll get that going on.  It wouldn't take 5 minutes.


NOTE:  I fixed up XM's post about fine ales the other day.


---
Ummm...


Still forgetting something...  Be prepared for another email...


[Back to top](#top)

**Comment by KVonGit** [comment]
Hey, @DarkLizerd !

Will you replace this first line with the second line, please?

It's line 12 in the script for **enter portalXM CMD**

To be removed:
```C++
else if (ListContains(ScopeVisible(), mindmelt portal)) {
```


To be replaced:
```C++
if (ListContains(ScopeVisible(), mindmelt portal)) {
```

![image](https://user-images.githubusercontent.com/30656341/30146347-15666022-935d-11e7-964b-8612d3fd67a1.png)


---
You won't be able to enter XM's portal, otherwise.

I fixed it, but didn't commit afterwards.

Sorry!


---
Alternatively, I can fix it whenever you get done.

If you're on a roll, don't stop!!!

**Comment by DarkLizerd** [comment]
Mainly, this was to get the latest updated version. What I'm planning to add will be to my copy, then just block it in the next time I get it. 
But I'll make your update to what I add...

**Comment by KVonGit** [comment]
>Current version: 0.0.67
(or is it???)


Oh, I just saw this  part.  Yes, sir, it is 0.0.67.

**Comment by KVonGit** [comment]
PS 

Don't click the DumbRH walkthrough if kvLib is loaded.  In fact, I don't start a walkthrough until I'm in the hub.  Otherwise, you've got a 50/50 shot of an error while my library is doing it's thing alongside all the alerts, hints, quests and such initializing.

**Comment by DarkLizerd** [comment]
I added that part, along with the "quick notes" to keep track of the latest version, and as a note for what is planed while it is checked out.
(cleared when checked back in)

**Comment by KVonGit** [comment]
Checking it out.

(The 200th Commit will be coming very shortly!)

**Comment by KVonGit** [comment]
version 0.0.70

added Warp Zone to hub (testing wall for XM)

**Comment by KVonGit** [comment]
0.0.71

Changed my mp3 to an embedded text file.

---
Use the warp zone.

---
It's nap-time.

The game is now checked back in.

---
PS

I'm too lazy to edit the version number at the top of this thread every time.  (Sorry!)

**Comment by KVonGit** [comment]
Check-check-check-check....

Checking it out!

**Comment by KVonGit** [comment]
Checked back in.

Added testing stuff for XM.

You'll see the two additions in the hub from the beginning.

---
PS

When everything is moving around, and I mean the panes and everything, just press ENTER.

...and enter TRAVEL to exit the wallOfText rooms.

This is totally prototype stuff.  Just playing with the CSS to make the text move around.  Added the other stuff as a placeholder way to get there.  Improvised the text and the encoded message and the seven digit code, too!

---
Also, I didn't know what DL or HK would do when Pix did the new thing he can do... So I just didn't write them into it.  Again, though.  This is not even a draft.  XM will be totally changing ALL of this, so he can add people into the circle if they'd like that, I'm sure.

0.0.72 (I think.  It says the time and date in the year published of each game too, so that's another way to keep track...)


...yep, 0.0.72.  I just checked.

**Comment by DarkLizerd** [comment]
I uploaded the new game start called Pixie Quest...
If everyone likes it, just add it in as the start...

So... after all this madness... when will we get back to the original group project???

**Comment by KVonGit** [comment]
> I uploaded the new game start called Pixie Quest...
> If everyone likes it, just add it in as the start...

Checking out right now.

> So... after all this madness... when will we get back to the original group project???

I think XM is working his last bit in, in between crazy back-to-school type activities, then, we just decide on the ending.

The easiest thing to do would be having a DL clone grab the shard Pixie acquires in Crate Mart, and run away with it to kick off the group project.

**Comment by KVonGit** [comment]
Holy shit, DL!

That's really good!

---
>The group you work with said they almost had the project finshed, buy it needed your final run through and your input on a problem they were having. They figured your input would solve the problem.

The only thing I'd suggest changing is this.  If we say A PROBLEM, we'll have to come up with a real problem for Pixie to solve!


How about something like this:

> The group you work with said they almost have the project finished, buy they need your input on a few problems they're having involving the forum (as well as the hub). 
> 
> (They figure you could just swoop in and save the day.)

**Comment by KVonGit** [comment]
We can even add a "KILL THE WUMPUS AND FEAST ON ITS HEART" Quest in the beginning!

**Comment by KVonGit** [comment]
As soon as we have XM's new sequence (and I'm not rushing art), we'll put it all together, and everyone can play through the whole thing.  (I think it will end up being about 200 turns in all.)

At that point, each of us is bound to have a little input that will make all of our pieces fit together well.

Then, we just decide on the ending, and maybe a few little details, and FIN.  Done. The end.

Also, most of the game ties into the group project (or what I imagine the project will be anyway), especially DL's parts, so we've kind of technically been working on the group project this whole time.

I know that I've learned MANY tricks during the creation of my bits that I'll be using in the group project.

Plus, we got a better handle on what to expect from each other.

PLUS, Pixie will know what to expect out of us, too.  (And I just know he's gonna shit when he sees this!  Let alone while he's playing!  We've got three totally different things going on, with the underlying premise of the group project lurking around in here somewhere!)

---
We'll start a new Issue once we've got it all together for the first time, and that, my friends, will be the final issue!

**Comment by KVonGit** [comment]
I vote we start the game with this, then, upon exiting the elevator, we drop the title splash and start him in the hub, just like we've had it.

**Comment by KVonGit** [comment]
>You think back to the last reflective surface you came across....

Ha!  Just found this!

I wish I could write like you two!  Aargh!

Oh well... I'll just fool around with the display settings and the game mechanics instead!

**Comment by DarkLizerd** [comment]
This is what I was talking about in the VERRRRYYYY beginning...
Some people can write the story, someone else does the "underwork" to make it happen...
You guys know Quest and Quest programming... (I bet I could still beat you if we did this in Basic)
But I can story write...
I can do basic Quest, but only as far as it is similar to Basic. ( but forget spelling, we are not friends!!!)

The impression I get in the hub, is that we are working on the side project (this one) but not the main one. (My character even says so.) 
That may change the tone of what everyone is doing. Or, are our characters really working on the main project, and this "project" is just us working on the main project? 
(It sounds strange when you are in your own game...)

**Comment by KVonGit** [comment]
>The impression I get in the hub, is that we are working on the side project (this one) but not the main one. 

Yep.  You got it right!

**Comment by KVonGit** [comment]
The game is full of vague profundities.

None of the old classics explain EVERYTHING.

And, when sequels fill in the blanks, it usually messes the whole shooting match up!

**Comment by KVonGit** [comment]
Okay, I just checked the game back out to edit it.

Let me know if anyone gets an inclination to edit it.

---
Sorry if I burnt you guys out on Quest, by the way.

I get a little crazy.

**Comment by DarkLizerd** [comment]
You gona plug in my start???
(go for it!)

**Comment by KVonGit** [comment]
Already on it!

**Comment by KVonGit** [comment]
...if XM doesn't approve, we can easily revert.

...but I bet he digs it.

**Comment by KVonGit** [comment]
Just updated to 0.0.74!

- Added in DL's opening.

- Put BIG notes on the things I added in for XM to test out:  PLACEHOLDER FOR XM'S REAL OBJECT.

---
You guys think this needs to be changed now, too, right?

![image](https://user-images.githubusercontent.com/30656341/30315586-19b7482e-976a-11e7-8333-36a0fb6d5195.png)


Well, tell me after you play through to it.

**Comment by KVonGit** [comment]
Just check her back in.

It's nap-thirty.

---
Version 0.0.76

---
My **TODO**

- [ ] Add a mini-Quest with a message to decode, just to get PIxie warmed up for XM's swirling wall of text.
- [ ] Add dumb responses for every possible command.
- [ ] Combine all the start scripts I've got going on into as few as possible.
- [ ] Proofread
- [ ] Tell DL and XM how freaking AWESOME they are!!!
- [ ] Just delete the synopsis from the bit pictured in the above post.  (Or add something to make it like a 'welcome, Pixie!  Here's your mission!' message.)
- [ ] Thank the guys again.  (And apologize for all the emails, posts, threads, Issues, Comments, and all that stuff.)
- [x] Post this comment, shut down the PC, and get some sleep.
- [x] Say, "goodnight, Gracie."

Goodnight, Gracie!!!

**Comment by KVonGit** [comment]
Hey!

You two guys are freaking AWESOME!!!  (Just so you know.)

---
I really, really do want to thank you guys for everything!

(And I apologize for all the emails, posts, threads, Issues, Comments, and all that stuff.)

---
...and I'm checking the game out.

**Comment by KVonGit** [comment]
Checked it back in.

---
I added a little, but you can't tell, except for the warning at the beginning...

(I was actually working on the spells stuff for the ACTUAL group project instead. (Imagine that!))

---
I'm thinking of keeping the splash with the synopsis...

Might just leave the "For Pixie" line and delete the rest...

Thoughts?

---
I've got KV in the large cubed crate.

I'll have it check if all other quests are completed before Pixie can do anything in Crate Mart.  If he enters Crate Mart, the Cratesman will boot his ass back to the hub.

Once he's in, he'll find the shard, and he'll find us in the crates.

We'll be writing the game while he's playing it, and we'll be DICKS.

I actually plan to have RH and KV in the same crate, both working on their respective laptop (which Pixie can see the 'group project' we've been working on throughout the game on now -- or he can at least see a few lines of it anyway).  He'll have to figure out which one of them is actually writing the story (and I'm considering making that random, changing the attribute each time the crate is entered), then he'll have to switch off their computer when they're distracted (by the blizzle (foe shizzle, dizzle!)).

---
All along
it's all
just bricks in the wall.

**Comment by XanMag** [comment]
Just an update.  I know I've been a bit slow, but I can finish this part of mine (which seems small now in comparison!) no later than Saturday evening.  I'm so confident that I can finish that I am willing to really almost promise that it will be complete.  

Now, just as a recap because I am sure I am behind the times here...

My part in a tiny nutshell:
Pixie, largely through speaking to my NPC, must discover he has to complete some pretty remedial tasks for me.  Once complete I, as I have always done, offer him some far from appropriate recompense for his hard work.  In this case, I will give him some tacky, bejeweled certificate of appreciation.  The centerpiece of the certificate is a shard (which I believe is needed in some manner for end-game, right?).  The tasks he must complete are: 1) getting me my favorite beer, 2) creating a "dope-ass" beat, and 3) hacking into my word document and giving me needed feedback on my novel.  All of these things he must do (channeling my inner-Yoda, I am) while inside of my mind.  

There are VERY simple puzzles/info required for him to complete these so I don't think it is very difficult.

I haven't kept up appropriately with the specifics of your parts of the game, so I hope my plan fits nicely with your alls (yours alls?  yours all?).  I think it does, but let me know.  And FYI... I am completely done with #1 and #2.  I just need to solve the hacking part of my word document (I know KV has been all over the swirling wall of text).  If that is too messy, I can easily make that part less aesthetic and send Pixie to an external website to copy-paste.

**Comment by XanMag** [comment]
And for the record... I think this will turn out to be a pretty sweet game.  Merging the gifts we all have has worked out better than I thought and I look forward to adding Pixie to the mix in the next/ongoing group project!  =)

**Comment by KVonGit** [comment]
>a shard (which I believe is needed in some manner for end-game, right?)

No shards are required.  I just threw the shard in the crate from the jump to tie this into the group project.

I'm fine without the big ending, too (especially seeming how I've been staring at the blank screen for quite a while, but no text has appeared).  After playing through what we've all got, though, it kind of just stops the way it is now.

We could just have the wumpus bust in and take the shard when he returns to the hub with it.

# FIN

---
>There are VERY simple puzzles/info required for him to complete these so I don't think it is very difficult.

They're not difficult to you!  You wrote it!  It took me 30 minutes to figure out how to get into the beat lab, and I knew it was there!

---

>I haven't kept up appropriately with the specifics of your parts of the game, so I hope my plan fits nicely with your alls (yours alls? yours all?).

```you alls'```

...and it fits perfectly.

---
>I just need to solve the hacking part of my word document (I know KV has been all over the swirling wall of text). If that is too messy,

Boot up the version that's online now.

Get the blizzle.

USE the Warp Zone in the hub.

(Enter TRAVEL to get back out. (If we're using that 'wall of text', we need to add the hint to enter TRAVEL.))

Press F10, and do your walkthrough (testXM, or whatever).

Light the blizzle while in the bar.

Exit to hub.

USE Warp Zone.

Copy the encoded message (which is stupid once decoded).

Hover over the text that references the seven digit code to get the seven digit code.

USE the text scrambler.

Press 2 to decode.

Paste the message. 

Enter the seven digit number.

TRAVEL to return to the hub.

**Comment by KVonGit** [comment]
>I think this will turn out to be a pretty sweet game. Merging the gifts we all have has worked out better than I thought and I look forward to adding Pixie to the mix in the next/ongoing group project!

Word up!

**Comment by KVonGit** [comment]
Aargh!

I just thought of something cool (maybe)...

---
The game Pixie could catch up with the dreaming Pixie at the end?

Meaning:

---
Pixie gets the shard.  The WUMPUS busts in, takes it, and exits stage left, leaving an entrance to his cave.

Pixie follows.

We'll add one room as a segue into the first room of the game.

Pixie reaches the wumpus.

The wumpus beep-roars...

Pixie wakes up.

Circular plot.

---
Thoughts?


Is that better or worse than us in the crates, writing the game as he plays?

**Comment by KVonGit** [comment]
EDIT:

Pixie wakes up.

He realizes XM's certificate is still in his inventory, and that it still contains one of the:

# SHARDS OF &infin;


## THE BEGINNING

**Comment by KVonGit** [comment]
EDIT:

Pixie wakes up.

The whole gang (that's us) bursts into his bedroom.

---
"Pixie, Pixie!  PLEASE tell us you're still working on the group project!" says someone.

"Yeah, Pix," mumbles someone else.  "We tried to do it without you, but you should have SEEN what we ended up with!"

"Pure crap," the third of us chimes in.  "The player was a Quest programmer, of all things! Can you believe that?"

"There were some good bits," XM interjects.  "Gal was my favorite."

"I liked DL's parts," adds KV.

"I just had a dream that I was playing a game just like that!" you exclaim.  "Come to think of it," you continue, addressing each member of the group in turn, "you, and you, and you were there..."

"Wait a minute, guys," says DL, pointing towards something in your inventory.  "LOOK!!!"


That's when you see it.

XM's certificate is still in your inventory, and it still contains one of the:

# SHARDS OF ∞
## THE BEGINNING

**Comment by XanMag** [comment]
And you claim not to be a good writer or an idea guy!??!  That's bloody brilliant.  I love it!

**Comment by DarkLizerd** [comment]
KV... your idea reminded me of The Wizard of Oz...
...
And then the Scarecrow, Tinman, and the Lion burst into the room All making their comments about missing you...
(But we may need a few cameos of the characters scattered through out the game...)
Then when Toto bursts into the room and jumps into Pixie's lap, he spills his beer, perhaps even the one selected in the game, he wakes up (again) to the ending credits to...
The Wizard of Oz...

**Comment by KVonGit** [comment]
Well thank yaw, thank yaw.  (KV bowing.)

(You are talking about the last one, where we all bust in, right?)

If so:

---
I think everyone should write in what they'll say at the end.

---
I was thinking it was good... 

I still got a few goose-bumps the second time I read it.  

---
So, the plan is:

After everyone gets their Quests, points, and important objects (and hints, if applicable) all lined up, I'll make a change script (which I **just now** learned just means a damned IF...THEN... script (which I thought was called a 'conditional statement', but whatever)) that checks a dictionary for all the accomplishments.

THEN the Cratesman won't throw Pixie out of Crate Mart, Pixie acquires the shard, TRAVELS back to the hub, Wumpus..., follow..., circular plot..., Wizard of Oz/Nightmare on Elm Street ending.

---
##### COMING SOON
#### FROM SOME OF THE GUYS WHO HELP YOU WRITE GAMES...
### ...AND FOR THE GUY WHO MAKES IT ALL POSSIBLE
## GRAB YOUR ID, BECAUSE YOU'LL NEED IT TO PLAY:
# PIXIE'S QUEST

**Comment by KVonGit** [comment]
I was writing that, and missed your post when it showed up, DL.

>your idea reminded me of The Wizard of Oz...

Yep, and Nightmare on Elm Street (Pixie's still got something from the 'dream'.)

>Then when [HK] bursts into the room, he spills [XM's] beer, perhaps even the one selected in the game, he wakes up (again) to

**Cave**
You have made your way through most of the Wumpus cavern and are nearing in on the most foul creature itself. You recheck your sword, sharp as it ever has been. Your shield: at the ready. Your armor does not have a single gap that would let the creature harm you. 


You can go [east]().

\> x sword
Your trusty blade.

Forged from the iron the holistic metallurgist extracted from the blood of your enemies.

It has a red glow emanating from it. (That means there's either a Grue nearby... or a WUMPUS!!!)

\> x shield
It's your jewel-encrusted (yet somehow lightweight) shield, gifted to you by Tanya.


\> x armor
Slicker than owl shit, lighter than air, more powerful than JavaScript...

Your armor suits you.

**Comment by KVonGit** [comment]
Checked it back in.

0.0.48 (?)

I think...  

(sighs)

---
30 seconds later...

Nope. 

0.0.86!


---
And THAT'S why I decided to check it back in!  It kept telling my objects didn't exist!  I was like, 'yuh-huh!'  Then I'd look, and yeah... I was forgetting to put kv_whatever!  

UNDO * 10

SAVE

COMMIT

---
Goodnight, and have a pleasant tomorrow!

**Comment by KVonGit** [comment]
The game is checked back in.  (I accomplished: making the game act right after saving and loading.  It only took about... oh... 5 hours!  (Who's the one who set up all the CSS, anyway? (Oh, yeah.  This guy!)))

---
Also added a PDF for your perusal.  (It's a transcript.)

---
That's the news, and I am outta here!

**Comment by DarkLizerd** [comment]
New idea on the ending... No, the Wumpus does not steal the crystal, it is too big... but...
(and may need a few sightings elsewhere just to introduce them...)
We need 3 objects, Planet Disk (got it), the green crystal (got it), and a ring-like device, 6 inches across, with 4 inch legs. (Who wants this to be found in their world???)
From what we know (maybe know), these devices combine to create a very powerful weapon... or a very powerful power source, something that could power all of the US or Europe forever!
What we don't know is that these are only part of the larger device... (but the Wumpus knows!)
So, while we have the devices on the desk, trying to fit them together, Pixie notices a red fox sitting the SW corner, defiantly looking out of place. It is just sitting there watching what we are doing. As Pixie goes to check it out, there is a bark from another corner. Pixie and everyone looks over at the red fox in the NE corner. when Pixie looks back at the fox in the SW corner, it is gone! Suddenly, a fox, wearing a black translucent cape, and wielding a 18 inch long rapier, and a green hat (picture a fox version of Robbin Hood) jumps up on the desk, flashing his rapier around, stating, "Ah, good, you have recovered the parts for my master! We'll relieve you of them and be gone!" Out of surprise, everyone has backed up from this display, and suddenly, 3 more foxes appear out of nowhere, grabs the devices, and bolts for the NW corner. The first fox jumps off the desk running right behind the other 3. You notice that they are all wearing the same translucent capes that makes them blend into the background. As they near the corner, 2 brown rabbits appear out of nowhere, with a 2 foot black hole behind them. From the way they appeared, it was as if they drop another cloak they were holding. The 4 foxes dive through the hole, followed by the 2 rabbits. It takes everyone a moment to figure out what happened... 
Including the shock of a rapier wielding talking fox!!!
... The Master? Talking foxes? What will we see next???

**Comment by KVonGit** [comment]
![image](https://vignette.wikia.nocookie.net/disney/images/4/42/Robin_Hood_Disney.jpg/revision/latest?cb=20160706144649)

![image](https://media.vanityfair.com/photos/577eae9ebcd0f7fe659906f0/16:9/w_1200,h_630,c_limit/bunnies-secret-life-of-pets-donnie-darko-watership-down.jpg)

**Comment by KVonGit** [comment]
Rabbits and foxes and Masters? (Oh my!)


---
Do the rabbits dig 'holes' through dimensions?

**Comment by KVonGit** [comment]
![image](https://i.pinimg.com/originals/aa/f8/7b/aaf87bbde0ecf04721c0f3301dcd35c9.jpg)

**Comment by KVonGit** [comment]
That came out of nowhere, DL!

---
I'm still trying to wrap my mind around it.

My initial reaction was 'no', but it has an 'I don't know, it sounds kinda cool' aftertaste....

---
I like Disney's Robin Hood and Watership Down, by the way.

I wasn't making fun.

**Comment by DarkLizerd** [comment]
The rabbits digging dimensional holes...
Didn't think of that... (but that could be an idea...)
More like they have a supply of "portable holes" that they can stick on a wall, and presto!!! Instant hole to the other side, or further... then pull it off from the other side and the hole goes away...

**Comment by KVonGit** [comment]
Tired of scrolling so much.

Closing this and moving it to  Issue #35 .

https://github.com/KVonGit/HVACR-specifications/issues/35

**Comment by KVonGit** [comment]
Tired of scrolling so much.

Closing this and moving it to  Issue #35 .

https://github.com/KVonGit/HVACR-specifications/issues/35
